saleor/graphql/product/filters.py

```diff
@@ -667,6 +665,13 @@ class Meta:
         model = ProductType
         fields = ["search", "configurable", "product_type"]
 
+    @classmethod
+    def filter_product_type_searchable(cls, queryset, _name, value):
```

Comment:
Would be good to cover this method

Reply:
This case is already tested by test_product::test_filter_product_types_by_custom_search_value, if I understand you comment properly.

Reply:
Sure, I agreed. I meant a test case to see if the correct queryset is returned. Pure unit test for new added method `filter_product_type_searchable`